### DISCUSSION TITLE
publish the package within the organization and without the organization name in the package name

### DISCUSSION THREAD

**Discussion by Rajesh6381** [community/community#141045] [npm]
Hi, when I try to publish a package to npm within an organization, it requires a package name like @example-org/example-package-name. However, I want to publish the package within the organization without including the organization name in the package name—just as example-package-name.

npm version - 10.7.0
node version - 20.13.1

**Comment by hant121** [comment]


**Reply by Rajesh6381** [reply]
There is no way to transfer the teams manually
